pkg/gather/clusterconfig/operators.go

```diff
@@ -88,7 +95,7 @@ func gatherClusterOperators(ctx context.Context, configClient configv1client.Con
 		relRes := collectClusterOperatorResources(ctx, dynamicClient, co, resVer)
 		for _, rr := range relRes {
 			records = append(records, record.Record{
-				Name: fmt.Sprintf("config/clusteroperator/%s-%s", co.Name, rr.Name),
+				Name: fmt.Sprintf("operator-resources/clusteroperator/%s-%s", co.Name, rr.Name),
```

Comment:
This is a topic for discussion. We already gather `clusteroperator` resource configs e.g in `config/clusteroperator/authentication.json`, but the resources the operators operate are something different and we will also likely have OLM operator resources in the future.